Back-End/controllers/admin_controllers/students_controllers.js

```diff
@@ -1,6 +1,26 @@
 const db = require('../../db');
 
+
+const students_details = (req, res) => {
+    const query = `SELECT * FROM users WHERE role_id = 3`;
```

Comment:
same as the query above

Reply:
Point taken, but I'm referring to a specific id in a table. How can I make that dynamic?